docs/Contributing.md

```diff
@@ -60,6 +60,20 @@ npm start
 open http://localhost:8000/
 ```
 
+### Static Docs Site
+
+Markdown and source code for the [static docs site](https://pricelinelabs.github.io/design-system/) is located in
+`docs/`. This [x0](https://www.npmjs.com/package/@compositor/x0) project requires Node 7+.
```

Comment:
If the `compositor` installation step is not needed, you probably won't need this sentence either.

Reply:
Even though it's installed locally, `x0` is still needed to run the docs locally. I added this sentence because using node <7 causes the following error because `async` support is missing.

```
/Users/craig/git/design-system/docs/node_modules/@compositor/x0/bin/cli.js:69
      .then(async html => {
            ^^^^^
SyntaxError: missing ) after argument list```

Reply:
@unscsprt , thanks for the elaboration of this Node.js version observation. If Node.js version 7+ has become a requirement then.

Question to all,
What do you guys think of using the [engines](https://docs.npmjs.com/files/package.json#engines) field in the `package.json`, to take advantage on a tiny bit of the npm `package.json` features?
> You can specify the version of node that your stuff works on:

Reply:
@unscsprt et al,

Seems like Node.js 7+ has become a requirement. What do you guys think of using the [engines](https://docs.npmjs.com/files/package.json#engines) field of the `package.json` to take a tiny bit of advantage on npm `package.json` features?
> You can specify the version of node that your stuff works on: